config/config_util.go

```diff
@@ -53,7 +53,6 @@ var (
 		"Log.QueryLogMaxLen":              {},
 		"Log.ExpensiveThreshold":          {},
 		"CheckMb4ValueInUTF8":             {},
-		"EnableStreaming":                 {},
```

Comment:
Shall we add it to here?
https://github.com/pingcap/tidb/blob/master/config/config.go#L781

Without this, if the old cluster use this configure, and upgrade a new version,
the tidb-server will not be able to run.

Reply:
I don't believe the config parser is strict by default - but there is an optoin to do it: https://github.com/pingcap/tidb/pull/9855

But yes, good point. I will add it to the deprecated list.

Reply:
I took a look. It's already there:

https://github.com/pingcap/tidb/blob/493eb45c75a8f9687d5bf9185e5e167aadc81012/config/config.go#L795